common/web/lm-worker/src/model-compositor.ts

```diff
@@ -233,7 +236,8 @@ class ModelCompositor {
       //        Whitespace is probably fine, actually.  Less sure about backspace.
 
       let bestCorrectionCost: number;
-      for(let matches of searchSpace.getBestMatches()) {
+      const SEARCH_TIMEOUT = this.testMode ? 0 : correction.SearchSpace.DEFAULT_ALLOTTED_CORRECTION_TIME_INTERVAL;
+      for(let matches of searchSpace.getBestMatches(SEARCH_TIMEOUT)) {
```

Comment:
For reference:

https://github.com/keymanapp/keyman/blob/30747f5600ccc0587eaa3f2b5ba300c43c3632d8/common/web/lm-worker/src/correction/distance-modeler.ts#L532-L545

The `0`-based timer kill switch (lines 539-540) is used by lower-level unit tests, but before this PR, there was no pathway to use it during integration tests - especially not during those that run in the browser.